### ISSUE TITLE
Crowdin Integration Meeting: Tuesday, May 22 8am Pacific

### ISSUE THREAD

**Issue by obensource** [nodejs/i18n#87]
# Let's meet!
We need to schedule a time to record our process for setting up the crowdin-github integration: https://github.com/nodejs/i18n/issues/77

Let's make it happen before the Collaboration Summit! I'm generally available in the afternoons/evenings PST. How about y'all?

**Comment by zeke** [comment]
How about Thursday May 17 at 4pm Pacific?

**Comment by maddhruv** [comment]
That would be 4:30 AM in India!
Even if I wake up this early, my roommate sleeps hard :pensive:

**Comment by lukaszewczak** [comment]
It will be 1:00am in Poland, so it will be very difficult to join :-), but as soon this will happen then better, I hope a recording of this meeting will be available online?

**Comment by zeke** [comment]
@obensource I know you said afternoons/evenings are best, but could you possibly make a morning work? 8am Pacific would be more likely to work for @lukaszewczak and @maddhruv.

http://everytimezone.com/#2018-5-16,180,b8jj

**Comment by obensource** [comment]
@zeke sure! The next 8am slot that works for me would be Monday the 21st. Stoked to make it work for everyone–thanks Zeke! 🙌

**Comment by zeke** [comment]
How about Tuesday the 22nd? If that doesn't fit I can move stuff around and make Monday work.

**Comment by maddhruv** [comment]
This could work fine for me!
Might even reach home

**Comment by zeke** [comment]
I think we may need to postpone this meeting as we don't yet have Crowdin connected to the org.

See https://github.com/nodejs/i18n/issues/77 for details on why we're blocked.

**Comment by lukaszewczak** [comment]
Yes, I agree, we should postpone this meeting.

**Comment by zeke** [comment]
This meeting is postponed pending consideration from the TSC on authorizing the Crowdin-GitHub integration. See https://github.com/nodejs/TSC/issues/544#issuecomment-390859048 -- that  meeting is scheduled for May 23.

**Comment by zeke** [comment]
Still waiting on TSC approval. Follow https://github.com/nodejs/TSC/issues/544#issuecomment-390859048 for updates.

**Comment by zeke** [comment]
This meeting didn't happen because we didn't get TSC approval of Crowdin in time. Closing with the hope that we'll resolve this at the [collaborator summit](https://github.com/nodejs/summit/issues/60) this week.
